cognite/extractorutils/uploader.py

```diff
@@ -175,11 +182,86 @@ def stop(self, ensure_upload: bool = True) -> None:
             ensure_upload (bool): (Optional). Call upload one last time after shutting down thread to ensure empty
                 upload queue.
         """
-        self.stopping.set()
+        self.cancelation_token.set()
         if ensure_upload:
             self.upload()
 
 
+@dataclass(frozen=True)
+class TimestampedObject:
+    payload: Any
+    created: Arrow
+
+
+raw_uploader_rows_queued = Counter(
```

Comment:
Should these be capitalized (ie, do we consider these constants)? My gut says so since we don't expect users (or ourselves) to overwrite the contents of these variables.